Realm/RLMUser.h

```diff
@@ -197,7 +197,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)removeWithCompletion:(RLMUserOptionalErrorBlock)completion;
 
 /**
- Permanently deletes this user from your MongoDB Realm app.
+ Permanently deletes this user from your Atlas Device Sync app.
```

Comment:
So should this not be '`Atlas App Services app`'?

Reply:
Thanks probably I want to use Atlas App Services in couple of other places